src/routes/news/containers/WithCustomNewsProvider.js

```diff
@@ -180,50 +177,28 @@ const withCustomNewsProvider = <
   R: {}
 >(
     refresh: (refreshProps: R, dispatch: Dispatch<StoreActionType>) => void
-  ): ((
-  Component: React.ComponentType<S>
-) => React.ComponentType<PropsType<S, R>>) => {
-  return (
-    Component: React.ComponentType<S>
-  ): React.ComponentType<PropsType<S, R>> => {
-    return class extends React.Component<
-      PropsType<S, R>,
-      {| selectedNewsType: string |}
-    > {
+  ): ((Component: React.ComponentType<S>) => React.ComponentType<PropsType<S, R>>) => {
+  return (Component: React.ComponentType<S>): React.ComponentType<PropsType<S, R>> => {
+    return class extends React.Component<PropsType<S, R>, {| selectedNewsType: NewsType |}> {
       static displayName = wrapDisplayName(Component, 'withCustomNewsProvider');
 
-      state = {
-        selectedNewsType: this.getAvailableNewsType()
-      };
-
-      listRef = null;
+      state = { selectedNewsType: this.getAvailableNewsType() };
 
-      getAvailableNewsType (): string {
+      getAvailableNewsType (): NewsType {
         const { cityModel, selectedNewsType } = this.props.innerProps || {}
         const { tunewsEnabled, pushNotificationsEnabled } = cityModel || {}
         let type = LOCAL
         if (selectedNewsType) {
           type = selectedNewsType
```

Comment:
No need for let if you return directly
```suggestion
          return selectedNewsType
```

Reply:
Done